pkg/util/label_test.go

```diff
@@ -317,12 +359,18 @@ func TestRemoveLabel(t *testing.T) {
 					},
 					"spec": map[string]interface{}{
 						"replicas": 2,
-					}}},
+					},
+				},
+			},
 		},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			RemoveLabels(tt.args.obj, tt.args.labelKey)
+			if tt.args.labelKey == "" {
+				RemoveLabels(tt.args.obj)
+			} else {
+				RemoveLabels(tt.args.obj, tt.args.labelKey)
+			}
```

Comment:
You've given me a good reminder. I feel that the previous test method is not good. Maybe we can change `labelKey` to `labelKeys`, so that we can test `labelKeys` of different lengths.

Like this:

```go
RemoveLabels(tt.args.obj, tt,args.labelKeys...)
```